pkg/cmd/pr/list/list.go

```diff
@@ -36,6 +36,7 @@ type ListOptions struct {
 	HeadBranch string
 	Labels     []string
 	Author     string
+	App        string
```

Comment:
Can we name this Go field something slightly more descriptive, like AppAuthor?

Reply:
https://github.com/cli/cli/pull/5180/commits/a572d21b3386e258f9466f494be865839e6244bf